onnxruntime/core/providers/cpu/ml/treeregressor.cc

```diff
@@ -10,60 +10,41 @@ ONNX_CPU_OPERATOR_TYPED_ML_KERNEL(
     TreeEnsembleRegressor,
     1,
     float,
-    KernelDefBuilder().TypeConstraint("T", DataTypeImpl::GetTensorType<float>()).MayInplace(0, 0), \
+    KernelDefBuilder().TypeConstraint("T", DataTypeImpl::GetTensorType<float>()).MayInplace(0, 0),
     TreeEnsembleRegressor<float>);
 
 ONNX_CPU_OPERATOR_TYPED_ML_KERNEL(
     TreeEnsembleRegressor,
     1,
     double,
-    KernelDefBuilder().TypeConstraint("T", DataTypeImpl::GetTensorType<double>()).MayInplace(0, 0), \
+    KernelDefBuilder().TypeConstraint("T", DataTypeImpl::GetTensorType<double>()).MayInplace(0, 0),
     TreeEnsembleRegressor<double>);
 
 template <typename T>
 TreeEnsembleRegressor<T>::TreeEnsembleRegressor(const OpKernelInfo& info)
     : OpKernel(info),
-      nodes_treeids_(info.GetAttrsOrDefault<int64_t>("nodes_treeids")),
-      nodes_nodeids_(info.GetAttrsOrDefault<int64_t>("nodes_nodeids")),
-      nodes_featureids_(info.GetAttrsOrDefault<int64_t>("nodes_featureids")),
-      nodes_values_(info.GetAttrsOrDefault<float>("nodes_values")),
-      nodes_hitrates_(info.GetAttrsOrDefault<float>("nodes_hitrates")),
-      nodes_truenodeids_(info.GetAttrsOrDefault<int64_t>("nodes_truenodeids")),
-      nodes_falsenodeids_(info.GetAttrsOrDefault<int64_t>("nodes_falsenodeids")),
-      missing_tracks_true_(info.GetAttrsOrDefault<int64_t>("nodes_missing_value_tracks_true")),
-      target_nodeids_(info.GetAttrsOrDefault<int64_t>("target_nodeids")),
-      target_treeids_(info.GetAttrsOrDefault<int64_t>("target_treeids")),
-      target_ids_(info.GetAttrsOrDefault<int64_t>("target_ids")),
-      target_weights_(info.GetAttrsOrDefault<float>("target_weights")),
       base_values_(info.GetAttrsOrDefault<float>("base_values")),
-      transform_(::onnxruntime::ml::MakeTransform(info.GetAttrOrDefault<std::string>("post_transform", "NONE"))),
+      post_transform_(::onnxruntime::ml::MakeTransform(info.GetAttrOrDefault<std::string>("post_transform", "NONE"))),
       aggregate_function_(::onnxruntime::ml::MakeAggregateFunction(info.GetAttrOrDefault<std::string>("aggregate_function", "SUM"))) {
   ORT_ENFORCE(info.GetAttr<int64_t>("n_targets", &n_targets_).IsOK());
 
+  std::vector<int64_t> nodes_treeids_(info.GetAttrsOrDefault<int64_t>("nodes_treeids"));
+  std::vector<int64_t> nodes_nodeids_(info.GetAttrsOrDefault<int64_t>("nodes_nodeids"));
```

Comment:
nit: These names should all be updated as the trailing '_' is reserved for members.